_posts/2021-01-29-status-update.md

```diff
@@ -0,0 +1,90 @@
+
+## Networking Device Support Enhancements
+
+[Ciaran][Ciaran] has been working on enhancing the [cisco_ios](https://github.com/puppetlabs/cisco_ios) module to include two new [Bolt tasks](https://puppet.com/docs/bolt/latest/writing_tasks.html) to backup and restore configurations. The PoC has been proven out and now we're just wrapping up on some final development tasks before moving on to more extensive testing. We are looking to make this as universal as we can across as many Cisco devices as we can and have some exciting developments in the pipeline for better testing against Cisco devices. More details soon...
+## Community Contributions
+
+On the community day we normally focus on all the wonderful contributions we receive in Github, but over the past couple of weeks we have started to look at our JIRA backlog. There are a lot of tickets there as you can imagine. We have been categorising them based on the module they are associated with and also closing off tickets that are no longer relevant or that have already been fixed.
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-apache#2112`][puppetlabs-apache-pr-2112]: "Use the correct matches in rspec", thanks to [ekohl][ekohl]
+- [`puppetlabs-apt#965`][puppetlabs-apt-pr-965]: "(RE-13415) Update GPG key in example", thanks to [e-gris][e-gris]
+- [`puppetlabs-java_ks#349`][puppetlabs-java_ks-pr-349]: "Fix keytool path handling", thanks to [chillinger][chillinger]
+- [`puppetlabs-postgresql#1227`][puppetlabs-postgresql-pr-1227]: "Set default PostgresQL version for FreeBSD", thanks to [olevole][olevole]
+- [`puppetlabs-stdlib#1154`][puppetlabs-stdlib-pr-1154]: "Drop facter.d facts needed for Facter < 1.7", thanks to [ekohl][ekohl]
+- [`puppetlabs-stdlib#1150`][puppetlabs-stdlib-pr-1150]: "stdlib::ensure: new fuction to cast ensure values", thanks to [b4ldr][b4ldr]
+- [`pdk#938`][pdk-pr-938]: "(MAINT) Update Travis/Appveyor to build `main` branch", thanks to [scotje][scotje]
+- [`puppet-modulebuilder#32`][puppet-modulebuilder-pr-32]: "Use Puppet 7 in development on Ruby 2.7+", thanks to [ekohl][ekohl]
+- [`puppet-modulebuilder#31`][puppet-modulebuilder-pr-31]: "Add a setter for release_name", thanks to [ekohl][ekohl]
+- [`puppetlabs_spec_helper#330`][puppetlabs_spec_helper-pr-330]: "Restructure PuppetLint rake tasks so they can be configurable", thanks to [nmaludy][nmaludy]
+
+Please swing by our [office hours on Monday](https://puppet.com/community/office-hours/) on Slack if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-apache`][puppetlabs-apache] (`5.9.0`)
+- [`puppetlabs-postgresql`][puppetlabs-postgresql] (`6.9.0`)
+- [`puppetlabs-iis`][puppetlabs-iis] (`7.2.0`)
+
+## Our Open Source Contributors
```

Comment:
The contributors on [lines 66-72](https://github.com/puppetlabs/iac/pull/185/files#diff-e8352d965ccd9240ebc2b60fb1cf24eb3febd14d1a035d54be8d7246f3368470R66-R72) won't actually show up here. Sorry, I should have spotted this earlier. 

Something like this perhaps:
```MD
## Our Open Source Contributors

  - [ekohl][ekohl]
  - [e-gris][e-gris]
  - [chillinger][chillinger]
  - [olevole][olevole]
  - [b4ldr][b4ldr]
  - [scotje][scotje]
  - [nmaludy][nmaludy]
```